config/env/development.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+   // If you want to debug regression tests, you will need the following which is also in the test config:
+   zapHostName: "192.168.56.20",
+   zapPort: "8080",
+   // Required from Zap 2.4.1. This key is set in Zap Options -> API _Api Key.
+   zapApiKey: "v9dn0balpqas1pcc281tn5ood1",
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 952566d140f87205f5c601b2ecfb4a4d -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Use of Hard-coded Credentials
_Low CWE-798_
A secret, such as a password, cryptographic key, or token is stored in plaintext directly in the source code, in an application's properties, or configuration file. Users with access to the secret may then use the secret to access resources that they otherwise would not have access to.



### How to fix
Potential mitigations include the following:
  1. For outbound authentication: store passwords, keys, and other credentials outside of the code in a strongly-protected, encrypted configuration file or database that is protected from access by all outsiders, including other local users on the same system. Properly protect the key (CWE-320). If you cannot use encryption to protect the file, then make sure that the permissions are as restrictive as possible [REF-7].

In Windows environments, the Encrypted File System (EFS) may provide some protection.

  2. For inbound authentication: Rather than hard-code a default username and password, key, or other authentication credentials for first time logins, utilize a "first login" mode that requires the user to enter a unique strong password or key.
  3. If the software must contain hard-coded credentials or they cannot be removed, perform access control checks and limit which entities can access the feature that requires the hard-coded credentials. For example, a feature might only be enabled through the system console instead of through a network connection.
  4. For inbound authentication using passwords: apply strong one-way hashes to passwords and store those hashes in a configuration file or database with appropriate access control. That way, theft of the file/database still requires the attacker to try to crack the password. When handling an incoming password during authentication, take the hash of the password and compare it to the saved hash.

Use randomly assigned salts for each separate hash that is generated. This increases the amount of computation that an attacker needs to conduct a brute-force attack, possibly limiting the effectiveness of the rainbow table method.

  5. For front-end to back-end connections: Three solutions are possible, although none are complete.

    * The first suggestion involves the use of generated passwords or keys that are changed automatically and must be entered at given time intervals by a system administrator. These passwords will be held in memory and only be valid for the time intervals.
    * Next, the passwords or keys should be limited at the back end to only performing actions valid for the front end, as opposed to having full access.
    * Finally, the messages sent should be tagged and checksummed with time sensitive values so as to prevent replay-style attacks.